packages/waku/src/lib/handlers/dev-worker-impl.ts

```diff
@@ -141,17 +124,19 @@ const loadEntries = async (config: ResolvedConfig) => {
 
 parentPort!.on('message', (mesg: MessageReq) => {
   if (mesg.type === 'render') {
-    handleRender(mesg);
-  } else if (mesg.type === 'buf') {
-    const controller = controllerMap.get(mesg.id)!;
-    controller.enqueue(new Uint8Array(mesg.buf, mesg.offset, mesg.len));
-  } else if (mesg.type === 'end') {
-    const controller = controllerMap.get(mesg.id)!;
-    controller.close();
-  } else if (mesg.type === 'err') {
-    const controller = controllerMap.get(mesg.id)!;
-    const err =
-      mesg.err instanceof Error ? mesg.err : new Error(String(mesg.err));
-    controller.error(err);
+    const bridge = new TransformStream({
+      transform(chunk, controller) {
+        if (chunk instanceof Uint8Array) {
+          controller.enqueue(chunk);
+        } else if (chunk instanceof ArrayBuffer) {
+          controller.enqueue(new Uint8Array(chunk, 0, chunk.byteLength));
+        } else {
+          controller.error(new Error('Unexepected buffer type'));
+        }
+      },
+    });
+
+    mesg.stream?.pipeThrough(bridge);
```

Comment:
This bridge is almost no-op, no?

Reply:
Could you clarify what a no-op is?

Reply:
I mean it doesn't do anything. "no operation".

Reply:
isn't it converting array buffer also to Uint8Array?

Reply:
As we no longer use post message for stream, we shouldn't need any conversion.